### ISSUE TITLE
Game changes the monitor resolution to 1024 x 768

### ISSUE THREAD

**Issue by ssspacez** [supertuxkart/stk-code#3123]
Game starts with 1024 x 768 resolution and changes the monitor(system) resolution to 1024 x 768. I tried changing to 1366 x 768+fullscreen but it fails to change, after quiting the game the monitor resolution stays 1024 x 768.

Info:

Game version: 0.9.2+dfsg-2, download from gnome-software

```
System: 
Ubuntu 17.10 x86_64 (Xubuntu)
DE: Xfce
WM: Xfwm4
Graphics:  Card: Intel 2nd Generation Core Integrated Graphics Controller
           bus-ID: 00:02.0
           Display Server: x11 (X.Org 1.19.5 )
           drivers: modesetting (unloaded: fbdev,vesa)
           Resolution: 1366x768@59.97hz
           OpenGL: renderer: Mesa DRI Intel Sandybridge Mobile
           version: 3.3 Mesa 17.2.8 Direct Render: Yes

CPU:       Dual core Intel Core i5-2410M (-HT-MCP-) 
           arch: Sandy Bridge rev.7 cache: 3072 KB
           flags: (lm nx sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx) bmips: 9178
           clock speeds: max: 2900 MHz 1: 2294 MHz 2: 2294 MHz 3: 2294 MHz
           4: 2294 MHz
```

## Additional information

[stdout.log](https://github.com/supertuxkart/stk-code/files/1726324/stdout.log)

**Comment by qwertychouskie** [comment]
Try 0.9.3 from here: https://launchpad.net/~stk/+archive/ubuntu/dev

**Comment by ssspacez** [comment]
I've just downloaded the snap and tested, it works. 

The issue is solved by upgrading to the 0.9.3 version, thank you.
